spec/acceptance/include_warning_spec.rb

```diff
@@ -1,29 +1,30 @@
 require 'spec_helper'
+require 'structured_warnings_helper'
 
 describe "Including is discouraged" do
   context "when including LightService::Organizer" do
     it "gives warning" do
       expected_msg = "including LightService::Organizer is deprecated. " \
                      "Please use `extend LightService::Organizer` instead"
-      expect(ActiveSupport::Deprecation).to receive(:warn)
-        .with(expected_msg)
 
-      class OrganizerIncludingLS
-        include LightService::Organizer
-      end
+      expect do
+        class OrganizerIncludingLS
+          include LightService::Organizer
+        end
+      end.to warn_with(StructuredWarnings::DeprecatedMethodWarning, expected_msg)
```

Comment:
These tests are great! We should remember to remove them when the actual removal takes place. 👍🏻

Reply:
Given this discussion https://github.com/adomokos/light-service/pull/236#discussion_r908968696 it's still to take a decision about the use of that library.

That said and considering that whatever implementation will be it will have a test: remembering to remove deprecation from code will bring a red CI. That should be a good post-it to remove the test too ;)